src/lib.rs

```diff
@@ -92,14 +92,22 @@ impl<L: LogLevel> Verbosity<L> {
         }
     }
 
-    /// Get the log level.
+    /// Get the log level, as defined by the `log` crate.
     ///
     /// `None` means all output is disabled.
     pub fn log_level(&self) -> Option<log::Level> {
         level_enum(self.verbosity())
     }
 
-    /// Get the log level filter.
+    /// Get the log level, as defined by the `tracing` crate.
+    ///
+    /// `None` means all output is disabled.
+    #[cfg(feature = "tracing")]
+    pub fn tracing_level(&self) -> Option<tracing::Level> {
```

Comment:
For tracing, is it more appropriate to return `Level` or `LevelFilter`?

Reply:
To be clear, I'm not saying "also" but "instead of"

Reply:
They're very easily convertible to each other, and the equivalent of [`filter_level`](https://github.com/rust-cli/clap-verbosity-flag/blob/299177b24e8f756d22c742101fde5465fa8ab992/examples/example.rs#L15) accepts both, so I don't think it matters much.  I chose `tracing::Level` over `tracing::level_filters::LevelFilter` because I guessed that, if either's path were to change in the future, it more likely would be the latter's.